ArcGISRuntimeSDKQt_CppSamples/Routing/RouteAroundBarriers/RouteAroundBarriers.qml

```diff
@@ -0,0 +1,208 @@
+// [WriteFile Name=RouteAroundBarriers, Category=Routing]
+// [Legal]
+// Copyright 2020 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Controls 2.2
+import Esri.Samples 1.0
+import QtQuick.Layouts 1.11
+
+Item {
+
+    readonly property int checkBoxPadding: 20
+
+    // add a mapView component
+    MapView {
+        id: view
+        anchors.fill: parent
+
+        ColumnLayout {
+            spacing: 0
+            Layout.alignment: Qt.AlignTop
+
+            Rectangle {
+                id: backBox
+
+                z: 1
+                Layout.alignment: Qt.AlignLeft
+                Layout.margins: 3
+                Layout.bottomMargin: 0
+                width: Qt.platform.os === "ios" || Qt.platform.os === "android" ? 200 : 300
+                height: childrenRect.height
+                color: "lightgrey"
+
+                GridLayout {
+                    id: grid
+                    rows: 4
+                    columns: 1
+                    rowSpacing: 10
+                    columnSpacing: 2
+                    anchors.horizontalCenter: parent.horizontalCenter
```

Comment:
generally our styling is to have first property be id, and second would be anchors. Then no further order beyond that. Please move this up to just below id declaration